Crear modelo de EstadoEspecies

jorgegonzac opened the pull request:
Se agregó la migración correspondiente a la nueva tabla estado especie,
tomando en cuenta la creación de un nuevo campo en la tabla especie que
apunta con una llave foránea a la tabla estadoespecie. Se agregó el
modelo correspondiente a estado especie.

jorgegonzac committed:
Crear modelo de EstadoEspecies

Se agregó la migración correspondiente a la nueva tabla estado especie,
tomando en cuenta la creación de un nuevo campo en la tabla especie que
apunta con una llave foránea a la tabla estadoespecie. Se agregó el
modelo correspondiente a estado especie.

chsalgado commented on Api/OMMFCM/app/database/migrations/2015_10_13_023718_crearTablaEstadoEspecie.php:
```diff
@@ -0,0 +1,56 @@
+		// Crear tabla de estados de especies
+		Schema::create('estadosEspecies', function(Blueprint $table)
+    	{
+	        $table->bigIncrements('idEstadoEspecie');
```
Por que el IdEstadoEspecie es Big?

jorgegonzac replied:
Porque tomé el ejemplo de la tabla de incidentes, la cual tiene un bigInt. Aunque creo que existe la posibilidad de que no se den de alta un número tan grande de especies. Lo cambio a INT?

chsalgado replied:
Si, un int es suficiente

chsalgado commented on Api/OMMFCM/app/database/migrations/2015_10_13_023718_crearTablaEstadoEspecie.php:
```diff
@@ -0,0 +1,56 @@
+		// Agregar atributo de estado a las especies
+		Schema::table('especies', function($table)
+		{
+		    $table->bigInteger('idEstadoEspecie');		    
```
Same

jorgegonzac committed:
Cambio Tipo dato

Se optó por cambiar la columna idEstadoEspecie por Tinyint en lugar de
bigint dada la posibilidad de que no existirá un gran número de
registros en esta tabla

chsalgado commented:
Aprobada
